Source/WebCore/css/CSSProperties.json

```diff
@@ -2223,6 +2223,10 @@
                     "value": "contain",
                     "status": "unimplemented"
                 },
+                {
+                    "value": "intrinsic",
+                    "status": "non-standard"
+                },
```

Comment:
Seeing this makes me realize that it's hard to spot patterns in these JSON format files because they are so vertical and wordy.

Reply:
Yeah, JSON would not be choice for this. 

The reason it is JSON still is that it is used by both the build and to drive https://webkit.org/css-status/. Whatever we change it to, we would need to either a) duplicate some of it in a JSON file, b) have some way to generate the JSON file from the new format for the server, c) make the new format something that can be parsed in the browser as well.

For this particular thing, the values array, we ideally would just get rid of it, and use the grammars for this information. We run into the same problem as above here.

All solvable.